### ISSUE TITLE
XHTML serialization error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [luisvt/oryx-editor#19]
```
AS-is
The diagram http://b3mn.hpi.uni-potsdam.de/server.php?resource=Review
contains a multi-instance activity with a line break in its title (AB\nBA).
The XHTML serialization is now corrupt.

Expected
Proper serialization
```

Original issue reported on code.google.com by `uar...@gmail.com` on 9 Nov 2007 at 2:06

**Comment by GoogleCodeExporter** [comment]
```
I can confirm this. When you use '&' in a task's name for example, you'll get a
parser error when you reload oryx. String encoding problem.
```

Original comment by `NicoPete...@gmail.com` on 2 May 2008 at 3:47
- Changed state: **Accepted**
- Added labels: **Priority-Critical**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
sorry, I cannot confirm the line break error, but the HTML encoding problem.
```

Original comment by `NicoPete...@gmail.com` on 5 May 2008 at 7:11

**Comment by GoogleCodeExporter** [comment]
Original comment by `NicoPete...@gmail.com` on 5 May 2008 at 8:34
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
fixed in revision #239
```

Original comment by `NicoPete...@gmail.com` on 6 May 2008 at 10:07
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `NicoPete...@gmail.com` on 23 May 2008 at 3:25
- Changed state: **Verified**
